cluster/src/main/java/org/apache/iotdb/cluster/log/manage/UnCommittedEntryManager.java

```diff
@@ -126,11 +128,11 @@ public void truncateAndAppend(List<Log> appendingEntries) {
         long len = after - offset;
         if (len < 0) {
             // the logs are being truncated to before our current offset portion, which is committed entries
-            // unconditional obedience to the leader's request. Maybe throw a exception here is better
-            offset = after;
-            entries = appendingEntries;
+            // Maybe throw a exception here is better
             logger.error("The logs which first index is {} are going to truncate committed logs",
                 after);
+//            throw new TruncateCommittedEntryException(appendingEntries.get(0).getCurrLogIndex(),
+//                offset);
```

Comment:
Please just decide one and do not leave commented code. Both are okay for me.

Reply:
OK